front/components/assistant/AssistantBrowser.tsx

```diff
@@ -368,6 +370,12 @@ export function AssistantBrowser({
         </ScrollArea>
       </div>
 
+      {isLoading && (
+        <div className="flex justify-center py-8">
+          <Spinner size="lg" />
```

Comment:
OOC why the colored spinner here and on line 291 the black and white one?

Reply:
we use big color spinner for tables loading, the small bw one is in the dropdown menu @Duncid